build/images/test/start_ovs_netdev

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source logging
+source daemon_status
```

Comment:
did you test this? I don't think `logging` and `daemon_status` are available

Reply:
actually ignore this, they are available in the Docker image
however, I don't think `source daemon_status` is required?

Reply:
I also think the script can be further simplified and we can remove `log_error` / `log_info` / `source logging`